internal/testutils/endpoint.go

```diff
@@ -16,9 +16,7 @@ limitations under the License.
 
 package testutils
 
-import (
-	"github.com/kubernetes-incubator/external-dns/endpoint"
-)
+import "github.com/kubernetes-incubator/external-dns/endpoint"
```

Comment:
or not putting it back (this is just an internal package, will add back brackets if there will be more than one import) , let's just get it merged